src/applications/disability-benefits/686c-674/config/chapters/674/student-last-term-information/studentLastTerm.js

```diff
@@ -1,38 +1,18 @@
+import cloneDeep from 'platform/utilities/data/cloneDeep';
 import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
 import { TASK_KEYS } from '../../../constants';
-import { genericSchemas } from '../../../generic-schema';
 import { isChapterFieldRequired } from '../../../helpers';
 import { buildAddressSchema, addressUISchema } from '../../../address-schema';
-
+import { report674 } from '../../../utilities';
 import { StudentNameHeader } from '../helpers';
 
-const { date, genericTextInput } = genericSchemas;
-
 const addressSchema = buildAddressSchema(false);
 
-export const schema = {
-  type: 'object',
-  properties: {
-    studentDidAttendSchoolLastTerm: {
-      type: 'boolean',
-    },
-    lastTermSchoolInformation: {
-      type: 'object',
-      properties: {
-        schoolName: genericTextInput,
-        schoolAddress: addressSchema,
-        dateTermBegan: date,
-        dateTermEnded: date,
-        classesPerWeek: {
-          type: 'number',
-        },
-        hoursPerWeek: {
-          type: 'number',
-        },
-      },
-    },
-  },
-};
+const lastTermSchema = cloneDeep(report674.properties.studentLastTerm);
+
+lastTermSchema.properties.lastTermSchoolInformation.properties.address = addressSchema;
+
```

Comment:
This is done a couple of times because there are two properties that occasionally need to get removed from the addressSchema involving military base. Preferred to create a copy of the schema and modify that instead of the one directly imported.